test-harness/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ata-mollusk-harness"
+version = "1.0.0"
+edition = "2021"
+license = "Apache-2.0"
+
+[lib]
+path = "src/lib.rs"
```

Comment:
nit: Is this needed? maybe we just need `crate-type = ["lib"]`

Reply:
Not needed; overly explicit. Your suggestion is applied in 53bdee7